full-service/src/json_rpc/json_rpc_request.rs

```diff
@@ -113,6 +113,14 @@ pub enum JsonCommandRequest {
         tombstone_block: Option<String>,
         max_spendable_value: Option<String>,
     },
+    build_multi_transaction {
```

Comment:
We need to chat with @sugargoat and @davidyhall about making multi-lay transactions the only build transaction API. Also need to discuss if we should change build_and_submit to this style as well.

Reply:
This is done. There is now just one endpoint supporting both APIs.